pilot.go

```diff
@@ -134,6 +137,74 @@ func (c *chanController) SpliceOut(chanPoint *wire.OutPoint,
 	return nil, nil
 }
 
+// FeeEstimate returns the total estimated fee needed for the autopilot to open
+// channels of the given sizes. Since we currently open each channel in a
+// separate tx, the combined fee estimate will just be the sum of the
+// individual ones.
+func (c *chanController) FeeEstimate(chanSizes []btcutil.Amount) (
+	btcutil.Amount, error) {
+
+	// Find all unlocked unspent witness outputs that satisfy the minimum
+	// number of confirmations required.
+	coins, err := c.server.cc.wallet.ListUnspentWitness(
+		c.minConfs, math.MaxInt32,
+	)
+	if err != nil {
+		return 0, err
+	}
+
+	// Since we currently open each channel in a separate tx, the combined
+	// fee estimate will just be the sum of the individual ones. We attempt
+	// to mimic the coin selection done in the wallet. This won't be
+	// totally accurate (coins can be selected differently when the
+	// channels are actually being opened), but this is just an estimate
+	// soooo ¯\_(ツ)_/¯
+	var totalWeight int64
+	currentCoin := 0
+	for _, amt := range chanSizes {
+		var weightEstimate input.TxWeightEstimator
+
+		// Channel funding multisig output is P2WSH.
+		weightEstimate.AddP2WSHOutput()
+
+		// Change output is a P2WKH output.
+		weightEstimate.AddP2WKHOutput()
+
+		satSelected := btcutil.Amount(0)
+		for currentCoin < len(coins) {
+			coin := coins[currentCoin]
+			currentCoin++
+
+			switch coin.AddressType {
+			case lnwallet.WitnessPubKey:
+				weightEstimate.AddP2WKHInput()
+			case lnwallet.NestedWitnessPubKey:
+				weightEstimate.AddNestedP2WKHInput()
+			default:
+				return 0, fmt.Errorf("Unsupported address: %v",
```

Comment:
Nit: returned error should start with lowercase.

Reply:
Fixed: https://github.com/lightningnetwork/lnd/pull/3017/commits/ca4fc621f3a8bd818f981b4262c251f357e1cc61